flask_wiki/templates/wiki/page.html

```diff
@@ -25,8 +25,10 @@ <h1>{{page.title}}</h1>
     </div>
     <div class="col-md-4">
         <div class="sticky-top pt-3">
-            <h5>{{ _('Table of Content') }}</h5>
+            {% if page.toc %}
```

Comment:
This 'if' doesn't work. I've tried a few things but I did not manage to get this condition right.